### PR TITLE
Address CS2101 UG Draft comments

### PR DESCRIPTION
**Pull request by YHWong20**
Made changes to address Ms Amy's comments regarding our UG draft.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## Introduction
 
+BinBash is a CLI-based **Inventory Management System** that is targeted at small retail business owners.
```
Comment (PureUsagi) [inline docs/UserGuide.md:5]: Very good introduction, its concise and comprehensive. Tells readers what BinBash is and what is our target audience. Its also written in a formal tone.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -20,61 +24,105 @@ Our long awaited first release adds everything you would expect of an inventory
...
-## Getting Started
+> :information_source: This is an info box that may appear in the guide. Any additional information about a feature will be detailed in sections like this one here.
+
+If you are new to BinBash (or new to command line-based applications in general), we highly recommend that you read through the rest of this guide sequentially, following the order of sections in this guide.
+In particular, we suggest that you take note of the [command format](#notes-on-the-command-format) used by BinBash, as it would provide you with a better understanding of how BinBash processes your commands.
 
-To run BinBash, ensure that your computer meets the following minimum system requirements:
-* Operating System: Windows, macOS, or Linux
-* Java `11` is installed. Refer to our [FAQ](#faq) for more details.
+If you consider yourself a power user, feel free to jump to the [Command Summary](#command-summary) section.
```
Comment (PureUsagi) [inline docs/UserGuide.md:70]: Tone is gives of a friendly and accepting vibe for readers. Well done!

File: docs/UserGuide.md [hunk 3]
```diff
@@ -20,61 +24,105 @@ Our long awaited first release adds everything you would expect of an inventory
...
+1. Ensure that you have Java `11` or above installed on your computer. Refer to our [FAQ](#faq) for more details.
 2. Head over to our [GitHub Page](https://github.com/AY2324S2-CS2113T-T09-2/tp/releases) and download the latest version of `BinBash.jar`.
 3. Move the downloaded `BinBash.jar` file to an empty folder of your choice. This folder will now serve as the home folder for BinBash.
 4. Open the terminal/command prompt for your system.
    1. If you're on Windows, press `Windows Key + R` and type in `cmd`. Press enter to launch the command prompt.
    2. If you're on Mac, click on `Launchpad` > `Other` > `Terminal`. Alternatively, click on the Spotlight icon in your menu bar, and type in `Terminal`.
    3. If you're on Linux, open the terminal in your Linux distribution.
 5. Navigate to the folder containing `BinBash.jar` using the terminal/command prompt.
-6. Type `java -jar BinBash.jar` into the command prompt, and press enter.
+6. Type `java -jar BinBash.jar` into the command prompt, and press enter. If the application has started successfully, you should see the following output:
```
Comment (PureUsagi) [inline docs/UserGuide.md:83]: Nice inclusion of the welcome screen to give reader a visual reference to ensure that they are on the right track to getting started.

**Review by PureUsagi — APPROVED** [review]
Overall the additions are well done.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -411,6 +517,16 @@ If not, refer to Oracle's [guide](https://docs.oracle.com/en/java/javase/11/inst
...
+**Q**: I want to modify my `items.txt` file. How should I format my items?<br>
+**A**: Firstly, we recommend that you take a backup of your current `items.txt` file before editing it.
+
+Then, open the file in any text editor of your choice (you can use `Notepad` on Windows). Feel free to add, modify or remove rows, but do ensure that they adhere to this format:
+```text
+ITEM_TYPE|ITEM_NAME|ITEM_DESCRIPTION|QUANTITY|ITEM_COST_PRICE|TOTAL_UNITS_PURCHASED|ITEM_THRESHOLD|ITEM_EXPIRATION_DATE|ITEM_SALE_PRICE|TOTAL_UNITS_SOLD|
+```
+
+If your item does not contain a certain attribute (e.g, no `ITEM_SALE_PRICE`), replace its value with a whitespace.
+
```
Comment (nur-haziq) [inline docs/UserGuide.md:529]: This is extremely useful! Perhaps we could shift this portion to right after the portion on saving and loading the data file, to cater to users who wish to edit the data file themselves.

File: docs/UserGuide.md [hunk 5]
```diff
@@ -161,40 +252,61 @@ Format: `add -op -n ITEM_NAME -d ITEM_DESCRIPTION -q ITEM_QUANTITY -s SALE_PRICE
...
+   ```text
+   -------------------------------------------------------------
+   Noted! I have added the following item into your inventory:
+    
+   [P][O] milk
+       description: to make cofee
+       quantity: 2
+       cost price: $1.30
+       expiry date: 03-10-2024
+   -------------------------------------------------------------
```
Comment (nur-haziq) [inline docs/UserGuide.md:287]: Good addition of the sample output. 👍

**Review by nur-haziq — APPROVED** [review]
Thanks for the detailed additions to the UG. 😄

**Pull request merged**